packages/core/src/ceramic.ts

```diff
@@ -175,11 +177,12 @@ export class Ceramic implements CeramicApi {
   public readonly dispatcher: Dispatcher
   public readonly loggerProvider: LoggerProvider
   public readonly pin: PinApi
+  public readonly index: IndexClientApi
   readonly repository: Repository
 
   readonly _streamHandlers: HandlersMap
   private readonly _anchorValidator: AnchorValidator
-  private readonly _indexing: DatabaseIndexAPI | undefined
+  private readonly _indexing: DatabaseIndexApi | undefined
```

Comment:
Can we hide the `DatabaseIndexApi` within the `IndexApi`?  Does the `Ceramic` class need both or can one abstract away the other?

Reply:
sorry, can you explain the concern a bit more?  Couldn't `LocalIndexApi` have an `init()` method which in turn calls `DatabaseIndexApi::init()`, and Ceramic's `_init()` can just call the `LocalIndexApi::init()`?

Reply:
That was the first version of the initialisation logic haha :) Maybe it is not a big deal to make `init` available after all.

Reply:
Okay, I feel it is better now.